### DISCUSSION TITLE
Upgraded from 15.4 to 16.9, missing template file report

### DISCUSSION THREAD

**Discussion by martin323232** [inventree/InvenTree#8617] [Q&A]
Hi,

just upgraded server to latest version. Getting this error: "Template file 'report/inventree_build_order_base.html' is missing or does not exist." Running docker on Ubuntu.
Looks like the file exist when using "docker exec" on container
![image](https://github.com/user-attachments/assets/e80a1687-387f-49c1-8508-9380113f30c0)

Anyone that knows how to fix the issue?

**Comment by SchrodingersGat** [comment]
Martin, please provide a full log output here.

**Comment by martin323232** [comment]
`Loading config file : /home/inventree/data/config.yaml
Python version 3.11.9 - /usr/local/bin/python
/root/.local/lib/python3.11/site-packages/allauth/exceptions.py:9: UserWarning: allauth.exceptions is deprecated, use allauth.core.exceptions
  warnings.warn("allauth.exceptions is deprecated, use allauth.core.exceptions")
192.168.1.10 - - [03/Dec/2024:05:56:04 +0000] "GET / HTTP/1.0" 302 0 "http://192.168.1.93:1337/settings/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:04 +0000] "GET /index/ HTTP/1.0" 200 23248 "http://192.168.1.93:1337/settings/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:04 +0000] "GET /js/dynamic/settings.js HTTP/1.0" 200 11833 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:04 +0000] "GET /js/dynamic/calendar.js HTTP/1.0" 200 1341 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:04 +0000] "GET /js/dynamic/nav.js HTTP/1.0" 200 9262 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /js/dynamic/permissions.js HTTP/1.0" 200 1202 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/part/?search=&offset=0&limit=25&starred=true&active=true&category_detail=true&location_detail=true&search_regex=true HTTP/1.0" 200 52 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/currency/exchange/ HTTP/1.0" 200 203 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/part/category/?search=&offset=0&starred=true&search_regex=true HTTP/1.0" 200 2 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/build/?search=&offset=0&overdue=true&part_detail=true&search_regex=true HTTP/1.0" 200 2 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/order/po/?search=&offset=0&limit=25&supplier_detail=true&outstanding=true&search_regex=true HTTP/1.0" 200 4370 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/order/po/?search=&offset=0&limit=25&supplier_detail=true&overdue=true&search_regex=true HTTP/1.0" 200 52 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/order/so/?search=&offset=0&limit=25&customer_detail=true&outstanding=true&search_regex=true HTTP/1.0" 200 52 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/part/?search=&offset=0&limit=25&low_stock=true&active=true&category_detail=true&location_detail=true&search_regex=true HTTP/1.0" 200 52 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/order/so/?search=&offset=0&limit=25&overdue=true&customer_detail=true&search_regex=true HTTP/1.0" 200 52 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/order/so/shipment/?shipped=false&search_regex=true HTTP/1.0" 200 2 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/stock/?search=&offset=0&in_stock=true&part_detail=true&ordering=-updated&updated_after=2024-11-03&location_detail=true&search_regex=true HTTP/1.0" 200 45781 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/part/?search=&offset=0&ordering=-creation_date&created_after=2024-11-03&active=true&category_detail=true&location_detail=true&search_regex=true HTTP/1.0" 200 21452 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:05 +0000] "GET /api/build/?search=&offset=0&active=true&part_detail=true&search_regex=true HTTP/1.0" 200 6652 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:05 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /part/792/ HTTP/1.0" 200 74946 "http://192.168.1.93:1337/index/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /js/dynamic/calendar.js HTTP/1.0" 200 1341 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:08 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /js/dynamic/nav.js HTTP/1.0" 200 9262 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /js/dynamic/settings.js HTTP/1.0" 200 11833 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /js/dynamic/permissions.js HTTP/1.0" 200 1202 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /api/user/roles/ HTTP/1.0" 200 5383 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:08 +0000] "GET /api/currency/exchange/ HTTP/1.0" 200 203 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:09 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:09 +0000] "GET /api/stock/?search=&offset=0&cascade=1&in_stock=1&part=792&location_detail=true&part_detail=true&supplier_part_detail=true&search_regex=true HTTP/1.0" 200 2 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:09 +0000] "GET /api/part/category/tree/ HTTP/1.0" 200 21514 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:34 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /build/ HTTP/1.0" 200 21406 "http://192.168.1.93:1337/part/792/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /js/dynamic/permissions.js HTTP/1.0" 200 1202 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /js/dynamic/nav.js HTTP/1.0" 200 9262 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /js/dynamic/calendar.js HTTP/1.0" 200 1341 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /js/dynamic/settings.js HTTP/1.0" 200 11833 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:55 +0000] "GET /api/build/?search=&offset=0&part_detail=true&search_regex=true HTTP/1.0" 200 87346 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:58 +0000] "GET /build/76/ HTTP/1.0" 200 46036 "http://192.168.1.93:1337/build/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:58 +0000] "GET /js/dynamic/permissions.js HTTP/1.0" 200 1202 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:58 +0000] "GET /js/dynamic/calendar.js HTTP/1.0" 200 1341 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
172.18.0.4 - - [03/Dec/2024:05:56:58 +0000] "GET /auth/ HTTP/1.0" 200 0 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:58 +0000] "GET /js/dynamic/settings.js HTTP/1.0" 200 11833 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:58 +0000] "GET /js/dynamic/nav.js HTTP/1.0" 200 9262 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:58 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:56:59 +0000] "GET /api/build/line/?search=&offset=0&build=76&search_regex=true HTTP/1.0" 200 159921 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:57:23 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:57:40 +0000] "OPTIONS /api/report/print/?context=true HTTP/1.0" 200 628 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:57:42 +0000] "GET /api/report/template/?enabled=true&model_type=build&items=76&offset=0&limit=25 HTTP/1.0" 200 589 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
192.168.1.10 - - [03/Dec/2024:05:57:48 +0000] "GET /api/notifications/?read=false HTTP/1.0" 200 2 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
Bad Request: /api/report/print/
192.168.1.10 - - [03/Dec/2024:05:57:55 +0000] "POST /api/report/print/ HTTP/1.0" 400 95 "http://192.168.1.93:1337/build/76/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
`

**Comment by SchrodingersGat** [comment]
TBH that looks fine - probably what you saw was the automatic creation of default.template files. Is the server otherwise functioning correctly?

**Comment by martin323232** [comment]
Everything else seems to work, but the report issue still is there.
![image](https://github.com/user-attachments/assets/685d72ab-37ff-4ada-92e6-18ac3e0cb12a)
![image](https://github.com/user-attachments/assets/b0300879-8015-46fd-8a3b-0992eb122c53)

**Comment by martin323232** [comment]
Got it working now had to update 

`{% extends "report/inventree_build_order_base.html" %}`
to
`{% extends "report/inventree_report_base.html" %}`
